module/VuFindSearch/src/VuFindSearch/Backend/ProQuestFSG/Connector.php

```diff
@@ -64,9 +65,11 @@ class Connector extends \VuFindSearch\Backend\SRU\Connector
      * Constructor
      *
      * @param \Laminas\Http\Client $client An HTTP client object
+     * @param Config               $config ProQuestFSG config
      */
     public function __construct(
-        \Laminas\Http\Client $client
+        \Laminas\Http\Client $client,
+        protected Config $config
```

Comment:
Again, worth considering an array here instead of a Config object?

Reply:
This one seemed a bit sketchier because a) we're passing in the entire parsed config file, not one section, and b) it makes the method that constructs Connector look a little silly given how it creates the cache

```
  protected function createConnector()
    {
        $connector = new Connector($this->createHttpClient(), $this->proQuestFSGConfig->toArray() ?? []);
        $connector->setLogger($this->logger);
        if ($cache = $this->createConnectorCache($this->proQuestFSGConfig)) {
            $connector->setCache($cache);
        }
        return $connector;
    }
```

But I made the change because on balance if we're weeding out that old library, we shouldn't add it to more places.